config/webpack.config.js

```diff
@@ -210,7 +215,9 @@ function generateHtmlFiles(buildPath, scaffoldAssets) {
       filename: path.join(buildPath, rootUrl, 'index.html'),
       inject: false,
       scriptLoading: 'defer',
-      template: 'src/platform/landing-pages/dev-template.ejs',
+      template:
+        templateLayoutToDevTemplate[template.layout] ||
+        'src/platform/landing-pages/dev-template.ejs',
```

Comment:
The inline conditional is straightforward but might benefit from a helper function for readability and potential reuse.

```javascript
function getDevTemplate(templateLayout) {
  return templateLayoutToDevTemplate[templateLayout] || 'src/platform/landing-pages/dev-template.ejs';
}
...
template: getDevTemplate(template.layout),
...
```